pullrequests.go

```diff
@@ -30,6 +30,13 @@ const (
 	Synchronize PullRequestAction = "synchronize"
 )
 
+//MergeRequestResponse contains information about an attempted merge request.
```

Comment:
I think there needs to be a space after `//`. We don't run our static analysis tools on the vendor folder but you can change that in the YML file.